### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by yuliiapyrih**

### CODE REVIEW

File: src/isIsogram.test.js [hunk 1]
```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
...
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true 
+    if word is an isogram`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should return true
```
Comment (dashaprotsko) [inline src/isIsogram.test.js:16]: You should add a test case for non-empty isogram words as well. For example, you can test with the word 'isogram' and expect it to be true.

File: src/isIsogram.test.js [hunk 2]
```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true 
+    if word is an isogram`, () => {
+    expect(isIsogram(''))
```
Comment (dashaprotsko) [inline src/isIsogram.test.js:12]: You should provide a valid isogram word for this test case, such as 'isogram'.

File: src/isIsogram.test.js [hunk 3]
```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
...
+      .toBeTruthy();
+  });
+
+  it(`should return true 
+    if word is empty`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should be case-insensitive`, () => {
```
Comment (dashaprotsko) [inline src/isIsogram.test.js:22]: Consider using a more descriptive test case. For example, you can test with the word 'CaseInsensitive' and expect it to be true.

File: src/isIsogram.test.js [hunk 4]
```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
...
+    expect(isIsogram('Adam'))
+      .toBeFalsy();
+  });
+
+  it(`should return true
```
Comment (dashaprotsko) [inline src/isIsogram.test.js:27]: The test description should be 'should return false if word is not an isogram'.

**Review by dashaprotsko — APPROVED** [review]
Little fixes are excepted
